charts/paperless/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for Paperless.
+
+image:
+  repository: thepaperlessproject/paperless
+  pullPolicy: IfNotPresent
+  tag: latest
+  args: "['gunicorn', '-b', '0.0.0.0:8000']"
+
+strategy:
+  type: Recreate
+
+# See the following files for supported environment variables
+# https://github.com/the-paperless-project/paperless/blob/master/docker-compose.env.example
+# https://github.com/the-paperless-project/paperless/blob/master/paperless.conf.example
+# Be sure to set PAPERLESS_OCR_LANGUAGES as a minimum (See https://github.com/the-paperless-project/paperless/blob/master/docker-compose.yml.example)
```

Comment:
You can leave a default for this environment variable and remove this comment. The user can override it